.github/workflows/build-branch.yml

```
@@ -65,45 +67,34 @@ jobs:
-      - name: Get production token
-        id: token
-        run: |
-          # Get token from secrets, or use a placeholder for pull requests
-          # This ensures PRs can build, but won't have a valid token
-          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
-            echo "production_token=placeholder-token-for-pr" >> "$GITHUB_OUTPUT"
-          else
-            echo "production_token=${{ secrets.NOTIFIER_API_TOKEN }}" >> "$GITHUB_OUTPUT"
-          fi
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)